elements/lisk-p2p/src/p2p.ts

```diff
@@ -131,11 +131,14 @@ const BASE_10_RADIX = 10;
 export const DEFAULT_MAX_OUTBOUND_CONNECTIONS = 20;
 export const DEFAULT_MAX_INBOUND_CONNECTIONS = 100;
 export const DEFAULT_OUTBOUND_SHUFFLE_INTERVAL = 300000;
+export const DEFAULT_PEER_PROTECTION_FOR_NETGROUP = 0.034;
 export const DEFAULT_PEER_PROTECTION_FOR_LATENCY = 0.068;
 export const DEFAULT_PEER_PROTECTION_FOR_USEFULNESS = 0.068;
 export const DEFAULT_PEER_PROTECTION_FOR_LONGEVITY = 0.5;
 export const DEFAULT_MIN_PEER_DISCOVERY_THRESHOLD = 100;
 export const DEFAULT_MAX_PEER_DISCOVERY_RESPONSE_SIZE = 1000;
+export const SECRET_LENGTH = 4;
+export const DEFAULT_SECRET = randomBytes(SECRET_LENGTH).readUInt32BE(0);
```

Comment:
We don't have to export this two values.

Reply:
`DEFAULT_SECRET` is used in some unit tests where a real 32 bit entropy secret was required. I will unexport `SECRET_LENGTH`